app/components/ClosingBakerPoolNotification.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Notification from './Notification';
+import { NotificationLevel } from '~/features/NotificationSlice';
+import Button from '~/cross-app-components/Button';
+
+interface Props {
+    onClose(): void;
+    accountName: string;
+}
+
+/**
+ * A notification to be shown when an account is delegating to a baker pool
+ * that is going to close in the near future.
+ */
+export default function ClosingBakerPoolNotification({
+    onClose,
+    accountName,
+}: Props) {
+    return (
+        <Notification
+            className="flexColumn alignCenter"
+            level={NotificationLevel.ClosingBakerPool}
+            onCloseClick={onClose}
+        >
+            <div className="flex alignCenter">
+                The target pool of the following account will close soon.
+            </div>
+            <div className="flex alignCenter mT10">{accountName}</div>
```

Comment:
Maybe make this bold to make it more obvious that it is an account name :upside_down_face: